p5/core/font.py

```diff
@@ -109,7 +101,7 @@ def text_font(font, size=10):
     :type font: PIL.ImageFont.ImageFont
 
     """
-    p5.renderer.font_family = font
+    p5.renderer.text_font(font, size)
```

Comment:
Instead of passing in a default size of 10, we should probably use the current font size.

Reply:
Yes, fixed it.

Reply:
Thanks! Looks like the `size` parameter is being ignored by vispy's `text_font`, could you also do a quick fix for that as well?